web/js/jquery.jmri.js

```diff
@@ -510,6 +510,7 @@
                     jmri.socket.send("power", {});
                 } else {
                     $.getJSON(jmri.url + "power", function (json) {
+                    	if ($.isArray(json)) json=json[0]; //unwrap array                   	
                         jmri.power(json.data.state);
```

Comment:
The indentation differs for these two lines. It would be good to fix it before merge of this PR. It makes the code easier to read.